### DISCUSSION TITLE
ChaiNNer issue for converting models via model conversion guide

### DISCUSSION THREAD

**Discussion by DuckersMcQuack** [upscayl/upscayl#1032] [Q&A]
This is the issue i'm getting in chaiNNer 0.21.0 when loading the node from the guide.

![image](https://github.com/user-attachments/assets/8bdede63-bb10-4849-88ec-f3918f7b3b39)

**Comment by aaronliu0130** [comment]
Please do step 1, installing dependencies, of the guide first.
